src/main/java/org/zalando/nakadi/service/subscription/state/ClosingState.java

```diff
@@ -41,7 +41,7 @@ public void onExit() {
         } catch (final NakadiRuntimeException | NakadiBaseException ex) {
             // In order not to stuck here one will just log this exception, without rethrowing
             getLog().error("Failed to transfer partitions when leaving ClosingState", ex);
-            TracingService.logErrorInSpan(TracingService.activateSpan(getContext().getCurrentSpan(), false),
+            TracingService.logErrorInSpan(getContext().getCurrentSpan(),
```

Comment:
user will definitely not need it, it is quite internal exception, which we could track in the logs